subprojects/docs/src/samples/userguide/files/inputFiles/build.gradle

```diff
@@ -1,28 +1,26 @@
-task compile(type: JavaCompile)
```

Comment:
We cannot remove this line. The build will fail without it since there would be no compile task to configure.